python/lsst/ts/m2com/mock/mock_control_closed_loop.py

```diff
@@ -260,8 +259,45 @@ def load_file_lut(self, filepath):
                 self._lut[name] = np.float64(data)
                 if name == "F_E":
                     self._lut["lutInAngle"] = np.float64(data.keys())
-            else:
-                self._lut[name] = np.loadtxt(filepath / lut)
+
+        self._lut["temp_inv"] = self._calc_temp_inv_matrix()
+
+    def _calc_temp_inv_matrix(self):
+        """Calculate the temperature inversion matrix.
+
+        Based on the "LSST_M2_temperature_sensors_20171102.pdf". The sensor map
+        is at "doc/figure/temperature_sensor_map.jpg".
+
+        Radial gradient: T(r) = Tr * r / R
+        X gradient: T(x) = Tx * x / R = Tx * r * cos(theta) / R
+        Y gradient: T(y) = Ty * y / R = Ty * r * sin(theta) / R
+        Uniform bias: T(u) = Tu
+
+        The matrix is: [r/R, r/R * cos(theta), r/R * sin(theta), 1], and we
+        have: matrix * [Tr, Tx, Ty, Tu].T = [T1, T2, ..., T12].T
+
+        Returns
+        -------
+        `numpy.ndarray`
+            Temperature inversion matrix.
+        """
+
+        # Radius of mirror in inch
```

Comment:
wow! inches! for real? Any chance you can convert this to mm or cm? I think it is quite dangerous to mix and match imperial and metric values.. We use metric everywhere..

Reply:
Changed the unit to mm in commit: [c313c57](https://github.com/lsst-ts/ts_m2com/pull/14/commits/c313c57ab18054e2f5878d1c68d5dcd545f4ae34). Thanks!